src/common/AndroidUtils.ts

```diff
@@ -730,7 +743,7 @@ export class AndroidSDKUtils {
     private static isEmulatorAlreadyStarted(emulatorName: string): boolean {
         const findProcessCommand =
             process.platform === WINDOWS_OS
-                ? `tasklist /V /FI "IMAGENAME eq qemu-system-x86_64.exe" | findstr "${emulatorName}"`
+                ? `wmic process where "CommandLine Like \'%qemu-system-x86_64%\'" get CommandLine  | findstr "${emulatorName}"`
```

Comment:
On Windows we should use `wmic` instead of `tasklist` b/c it comes pre-installed since XP, it executes much faster that `tasklist`, and it is more powerful.